Makefile

```diff
@@ -18,3 +18,9 @@ ndsfuse.o: ndsfuse.c ndsrom.h tree.h
 
 clean:
 	rm -rf treetest ndstest ndsfuse *.o
+
+install:
+	cp ndsfuse ndstest treetest -t /usr/local/bin
```

Comment:
I don't think ndstest and treetest need to be installed. They are just test programs, not something useful to users.

Reply:
Alright, then we can just get rid of them.